soccer/gameplay/plays/offense/two_side_attack.py

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Are you trying to change anything in two_sided_attack? If not please revert changes made to this file.